### DISCUSSION TITLE
Duda ejercicio 2 tp2

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#60] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023590/0d9d61d3-2534-444e-b613-c3082452718d)
En este ejercicio lo que se estaría pidiendo sería algo como:
```
"ingrese un numero"
235
imprime 10 (es decir 2+3+5)
```
¿sería así no?

**Comment by martinvilu** [comment, marked as answer]
Correcto
